go/pkg/client/example_input_table_test.go

```diff
@@ -81,17 +81,28 @@ func Example_inputTable() {
 		return
 	}
 
-	// Now, we take a snapshot of the outputTable to see what data it currently contains.
-	// We should see the new rows we added, filtered by the condition we specified when creating outputTable.
-	outputRec, err := outputTable.Snapshot(ctx)
-	if err != nil {
-		fmt.Println("error when snapshotting table", err.Error())
-		return
-	}
-	defer outputRec.Release()
+	for {
```

Comment:
I'm worried about an infinite loop. At the very least, some sort of limiter. @rcaudy, I'm surprised that upon completion of `AddTable` that the data might not be available for snapshotting?